scripts/inference.py

```diff
@@ -73,6 +73,8 @@
 
 if args.compile:
     print("compiling model")
+    # Bug with kv-cache in PT2.1
+    torch._inductor.config.joint_graph_constant_folding = False
```

Comment:
if we have an issue filed we should link to it

Reply:
we don't rn, but as soon as we do I'll file it